-1

Tested at: 0e5820a741ef7c44b1f073bb4d48bcb0326c5477


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/dfcd2967aa91a8c91bc0cd5a2e0cf57300526e8d
https://github.com/cms-sw/cmssw/commit/b4980caf9657d4242f7e2e7a8ff4e9ba3838a16e
https://github.com/cms-sw/cmssw/commit/3303c8e143615ab824a0899a10273112a8c85ebe
https://github.com/cms-sw/cmssw/commit/734d41830c7a18998e87810ff17cd89550645b59
https://github.com/cms-sw/cmssw/commit/3e2476adb4864109e6464769b4fa7c0351569e56
https://github.com/cms-sw/cmssw/commit/4a2a37dc9eee5bdcd201980cbc3ed3f589b010b7
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21357/24508/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21357/24508/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21357/24508/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_10_0_X_2017-11-17-1100'
config/SCRAM/GMake/Makefile.rules:2086: recipe for target 'src' failed
gmake: *** [src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/dfcd2967aa91a8c91bc0cd5a2e0cf57300526e8d
https://github.com/cms-sw/cmssw/commit/b4980caf9657d4242f7e2e7a8ff4e9ba3838a16e
https://github.com/cms-sw/cmssw/commit/3303c8e143615ab824a0899a10273112a8c85ebe
https://github.com/cms-sw/cmssw/commit/734d41830c7a18998e87810ff17cd89550645b59
https://github.com/cms-sw/cmssw/commit/3e2476adb4864109e6464769b4fa7c0351569e56
https://github.com/cms-sw/cmssw/commit/4a2a37dc9eee5bdcd201980cbc3ed3f589b010b7
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21357/24508/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21357/24508/git-merge-result

--- a/DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc
+++ b/DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc
@@ -241,12 +241,12 @@ void TrackBuildingAnalyzer::initHisto(DQMStore::IBooker & ibooker, const edm::Pa
     seedStoppingSource->setAxisTitle("Stopping reason",1);
     seedStoppingSource->setAxisTitle("Number of seeds",2);
 
-    histname = "StoppingSourceVsPhi_"+seedProducer.label() + "_";
+    histname = "SeedStoppingSourceVsPhi_"+seedProducer.label() + "_";
     seedStoppingSourceVsPhi = ibooker.bookProfile(histname+CatagoryName, histname+CatagoryName, PhiBin, PhiMin, PhiMax, 2, 0., 2.);
     seedStoppingSourceVsPhi->setAxisTitle("seed #phi",1);
     seedStoppingSourceVsPhi->setAxisTitle("fraction stopped",2);
 
-    histname = "StoppingSourceVsEta_"+seedProducer.label() + "_";
+    histname = "SeedStoppingSourceVsEta_"+seedProducer.label() + "_";
     seedStoppingSourceVsEta = ibooker.bookProfile(histname+CatagoryName, histname+CatagoryName, EtaBin, EtaMin, EtaMax, 2, 0., 2.);
     seedStoppingSourceVsEta->setAxisTitle("seed #eta",1);
     seedStoppingSourceVsEta->setAxisTitle("fraction stopped",2);
@@ -266,7 +266,7 @@ void TrackBuildingAnalyzer::initHisto(DQMStore::IBooker & ibooker, const edm::Pa
     numberOfTrajCandsPerSeedVsEta->setAxisTitle("Seed #eta", 1);
     numberOfTrajCandsPerSeedVsEta->setAxisTitle("Number of Trajectory Candidates for each Seed", 2);
 
-    histname = "StoppingSourceVsNumberOfTrajCandsPerSeed_"+seedProducer.label() + "_";
+    histname = "SeedStoppingSourceVsNumberOfTrajCandsPerSeed_"+seedProducer.label() + "_";
     seedStoppingSourceVsNumberOfTrajCandsPerSeed = ibooker.bookProfile(histname+CatagoryName, histname+CatagoryName, candsBin, candsMin, candsMax, 2, 0., 2.);
     seedStoppingSourceVsNumberOfTrajCandsPerSeed->setAxisTitle("Number of Trajectory Candidates for each Seed", 1);
     seedStoppingSourceVsNumberOfTrajCandsPerSeed->setAxisTitle("fraction stopped",2);

--- a/DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py
+++ b/DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py
@@ -30,5 +30,5 @@ pvMonitor = cms.EDAnalyzer("PrimaryVertexMonitor",
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase1Pixel.toModify(pvMonitor, EtaBin=26, EtaMin=-2.5, EtaMax=2.5)
-phase2_tracker.toModify(pvMonitor, EtaBin=26, EtaMin=-2.5, EtaMax=2.5)
+phase2_tracker.toModify(pvMonitor, EtaBin=41, EtaMin=-4.0, EtaMax=4.0)
 

--- a/DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py
+++ b/DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py
@@ -26,7 +26,7 @@ pvMonitor = cms.EDAnalyzer("PrimaryVertexMonitor",
    EtaMin         = cms.double(-2.5)
 )
 
-// same as above, should be in sync with cut used in Vertex finder...
+# same as above, should be in sync with cut used in Vertex finder...
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase1Pixel.toModify(pvMonitor, EtaBin=26, EtaMin=-2.5, EtaMax=2.5)

